src/apple/system.rs

```diff
@@ -434,8 +451,17 @@ impl Default for System {
     }
 }
 
+// Not supported on iOS.
+#[cfg(target_os = "ios")]
+fn get_io_service_connection() -> Option<ffi::io_connect_t> {
+    None
+}
+
 // code from https://github.com/Chris911/iStats
+#[cfg(target_os = "macos")]
 fn get_io_service_connection() -> Option<ffi::io_connect_t> {
+    use super::macos::ffi;
```

Comment:
I'm still thinking **hard** about this: wouldn't it be better to reexport the sub-`ffi` content into the "top" `ffi` to avoid having such local `use`s everywhere?

Reply:
I actually did this in the commit I'm about to push, but you're right.